bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/PerformanceConfiguration.kt

```diff
@@ -34,7 +34,7 @@ public class PerformanceConfiguration private constructor(public val context: Co
     public var logger: Logger? = null
 
     @JvmSynthetic
-    internal val spanEndCallbacks: MutableList<SpanEndCallback> = ArrayList()
+    internal var spanEndCallbacks: MutableList<SpanEndCallback> = ArrayList()
```

Comment:
This is not accessible to customers, and needs functions added for `addOnSpanEnd` and `removeOnSpanEnd`

```suggestion
    internal val spanEndCallbacks: MutableList<SpanEndCallback> = ArrayList()
```